components/style/stylist.rs

```diff
@@ -1683,7 +1686,9 @@ impl PerOriginCascadeData {
     }
 
     fn clear(&mut self) {
-        *self = Self::new();
+        self.element_map = SelectorMap::new();
```

Comment:
Yes, later additions to PerOriginCascadeData will want to use `clear()` on some members (e.g. the Bloom filter ones).